### ISSUE TITLE
TensorFlow 1.8 upgrade causes Keras CI build to stall

### ISSUE THREAD

**Issue by fchollet** [tensorflow/tensorflow#19038]
Here's a detailed description of the issue: https://github.com/keras-team/keras/issues/10100

**Summary:**

- Keras runs its CI tests on Travis
- Keras has two builds for its TensorFlow backend tests (Py 2.7 and Py 3.6)
- TF dependency is installed via `pip install tensorflow` (CPU version)
- A few days ago this started installing TF 1.8
- Both TF backend builds immediately started timing out after 25-50 min (high variance) with the error "No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.". Normally the TF tests take 17-18 min.
- No test started failing, this is entirely a build stalling issue.

**Reverting to TF 1.7 fixes the issue.** The exact cause of the stalling is completely unknown.

In addition, we know that:

- The failure is stochastic; sometimes one of the two TF builds (for Python 2.7 and 3.6) fails while the other passes. Most common case is that both fail, indicating that P(failure) is high.
- There are cases where TF Py 2.7 fails while 3.6 passes and cases where 3.6 fails while 2.7 passes, so the issue is not specific to a Python version.
- The failure is **not** due to an increase in the time taken by the tests. Disabling all large tests did not resolve the issue.

**Such stalling problems on Travis typically arise from either:**

- hanging process
- excessive RAM consumption leading to freezing execution

It is likely that TF 1.8 suffers from one of these two issues.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 44 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!

**Comment by ziky90** [comment]
This issue seems to me still very actual, see unsuccessful attempt to update tensorflow in keras tests in https://github.com/keras-team/keras/pull/10674 and https://github.com/keras-team/keras/pull/10929, they kind of "monkeyhacked" this by skipping tests that are having this problem in Keras.

I'm also stuck with `tensorflow==1.7.0` for my tests leveraging multiprocessing (running several tests leveraging keras in a row). I'm having exactly the same problem also with `tf.keras` since `tensorflow==1.8.0`.

I'd be for re-opening this issue!
